diff --git git/board/duovero/duovero_mux_data.h.orig git/board/duovero/duovero_mux_data.h
index 67639a7..8729a8d 100644
--- git/board/duovero/duovero_mux_data.h.orig
+++ git/board/duovero/duovero_mux_data.h
@@ -204,7 +204,7 @@ const struct pad_conf_entry wkup_padconf_array_non_essential[] = {
 	{PAD1_FREF_CLK3_REQ, M7},					/* safe mode */
 	{PAD0_FREF_CLK3_OUT, (M0)},					/* fref_clk3_out */
 	{PAD0_SYS_NRESPWRON, (M0)},					/* sys_nrespwron */
-	{PAD1_SYS_NRESWARM, (M0)},					/* sys_nreswarm */
+	{PAD1_SYS_NRESWARM, (PTU | IEN | M0)},				/* sys_nreswarm */
 	{PAD0_SYS_PWR_REQ, (PTU | M0)},					/* sys_pwr_req */
 	{PAD1_SYS_PWRON_RESET, (M3)},					/* gpio_wk29 */
 	{PAD0_SYS_BOOT6, (M0)},						/* sys_boot6 */
